modules/virt-example-vf-host-services.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc
+
+[id="virt-example-vf-host-services_{context}"]
+= Example: Node network configuration policy for virtual functions
+
+:FeatureName: Host network configuration for virtual functions
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+Update host network settings for Single Root I/O Virtualization (SR-IOV) network virtual functions (VF) in an existing cluster by applying a `NodeNetworkConfigurationPolicy` manifest.
+
+You can apply a `NodeNetworkConfigurationPolicy` manifest to an existing cluster to complete the following tasks:
+
+* Configure QoS host network settings for VFs to optimize performance.
+* Configure the host network bonding specification to determine network traffic distribution to bonded VFs.
+* Configure the host network miimon specification to determine the frequency of link state checking between physical functions (PFs) and VFs.
+
+[NOTE]
+====
+To update host settings for SR-IOV network VFs by using NMState in clusters that also have the SR-IOV Network Operator installed, you must set the `externallyManaged` specification to `True` in the `SriovNetworkNodePolicy` resource. For more information, see the _Additional resources_ section.
```

Comment:
In which SriovNetworkNodePolicy, you typically have several ones.  
Suggest to highlight that this must be set in the SriovNetworkNodePolicy managing the PF on which the the NNCP is operating (if this is the case)

Reply:
Agreed. It should be highlighted that any `SriovNetworkNodePolicy` CR managing the PF in question should set `externallyManaged: true`

Reply:
Thanks for the feedback. I have updated the note accordingly. Let me know if any other issues.

Reply:
I still don't understand the wording, the term " host settings for SR-IOV network VFs" is rather ambivalent.
Suggest the following instead


`To update host network settings for SR-IOV VFs using NMState on PFs which also are managed through the  SR-IOV Network Operator, the `externallyManaged` parameter in the relevant `SriovNetworkNodePolicy` resource must be set to `True`. For more information, see the _Additional resources_ section.`

Reply:
Thanks David, I've updated the content with your suggestion.